Cargo.lock

```diff
@@ -3179,6 +3203,7 @@ checksum = "6e63cff320ae2c57904679ba7cb63280a3dc4613885beafb148ee7bf9aa9042d"
 name = "sql"
 version = "0.1.0"
 dependencies = [
+ "avro-rs 0.7.0",
```

Comment:
it's a little surprising that we've got avro 0.6 and avro 0.7 in our dep tree now?

Reply:
That's because I'm hardcoding it to umanwizard/avro-rs. Once I land my avro-rs change that will no longer be the case.